packages/lesswrong/server/search/elastic/ElasticQuery.ts

```diff
@@ -135,39 +140,41 @@ class ElasticQuery {
     };
   }
 
-  private compileSimpleQuery(): QueryDslQueryContainer {
+  private compileSimpleQuery(): CompiledQuery {
     const {fields} = this.config;
     const {search} = this.queryData;
     const mainField = this.textFieldToExactField(fields[0], false);
     return {
-      bool: {
-        should: [
-          {
-            term: {
-              objectID: {
-                value: search,
+      searchQuery: {
```

Comment:
NB: Diff looks more complicated than it really is because most of it is just changing indentation levels